script.js

```diff
@@ -0,0 +1,49 @@
+function addZeroes(num) {
+    const stringNum = num.toString().split('.')[1]
+    const len = stringNum && stringNum.length > 2 ? stringNum.length : 2
+    return Number(num).toFixed(len)
+}
+
+$('#target').submit(function(e){
+    e.preventDefault()
+    const bill = $('.money').val()
+    const tip = $('.radio:checked').val()
+    const tipPercentage = (tip / 100) + 1
+    const totalAmountUnrounded = bill * tipPercentage
+        const totalAmountRounded = Math.round(totalAmountUnrounded * 100) / 100
+    const tipAmountUnrounded = totalAmountRounded - bill
+        const tipAmountRounded = Math.round(tipAmountUnrounded * 100) / 100
+
+        let finalTip = addZeroes(tipAmountRounded)
+        let finalBill = addZeroes(totalAmountRounded)
+
+    if(isNaN(bill) === true) {
```

Comment:
`isNaN(bill) === true` is equivalent to `isNaN(bill)`. It's best to get rid of the `=== true` because at this time it's redundant.